CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.5.0
+
+- Enforce three new lint rules:
+  - `[avoid_shadowing_type_parameters]`,
```

Comment:
From the preview, looks like these do not linkify--please fix.

Reply:
Oops, fixed.